apps/admin-gui/src/app/shared/components/vo-select-table/vo-select-table.component.ts

```diff
@@ -25,7 +25,10 @@ export class VoSelectTableComponent implements OnChanges, AfterViewInit, OnInit
   selection: SelectionModel<Vo>;
 
   @Input()
-  isVoAdmin: boolean;
+  showCheckbox = false;
+
+  @Input()
+  showRecent = false;
```

Comment:
I would suggest to delete this variables and replace it with
`@Input()
  displayedColumns: string[];`
This would make the component more reusable and easy to add new column.

Reply:
It would also reduce the amount of code in this component.